### PR TITLE
Regions performance improvements

### PR DESCRIPTION
**Pull request by nicholasrq**
This PR optimizes performance when working with a large amount of regions.

Previously it was almost impossible to work with 30+ regions on a single image. This optimization allow you to work with hundreds of regions without significant performance loss.

Tested on up to 350 regions on a large resolution image (2242x2082, 1.2MB jpeg). Tests show that LSF now can handle up to 700 regions of any type (rect, ellipse, poly, keypoint and brush).

### CODE REVIEW

File: src/regions/RectRegion.js [hunk 1]
```diff
@@ -15,6 +15,11 @@ import { LabelOnRect } from "../components/ImageView/LabelOnRegion";
 import { guidGenerator } from "../core/Helpers";
 import { AreaMixin } from "../mixins/AreaMixin";
 import { getBoundingBoxAfterChanges, fixRectToFit } from "../utils/image";
+import chroma from 'chroma-js';
+import { useRegionColors } from "../hooks/useRegionColor";
+import { AliveRegion } from "./AliveRegion";
+
+chroma;
```
Comment (hlomzik) [inline src/regions/RectRegion.js:22]: missed?

**Review by hlomzik — APPROVED** [review]

File: src/regions/RectRegion.js (hunk 1 above)
Comment (nicholasrq) [reply src/regions/RectRegion.js:22]: nah, was just observing the api autocompletion and forgot to remove it

File: e2e/tests/empty-labels.test.js [hunk 2]
```diff
@@ -186,8 +184,8 @@ const MULTIPLE_TYPE = "multiple";
   });
 });
 
-Scenario(`Consistency of empty labels`, async (I, LabelStudio, AtSidebar, AtImageView, AtAudioView) => {
-  const { annotations, config, data, result = annotations[0].result } = require("../examples/image-bboxes");
+Scenario(`Consistency of empty labels`, async ({I, LabelStudio, AtSidebar, AtImageView}) => {
+  const { annotations, config, data = annotations[0].result } = require("../examples/image-bboxes");
```
[the remark this answers is not in the document]
Comment (nicholasrq) [reply e2e/tests/empty-labels.test.js:188]: oops, i was supposed to remove the result as it's unused during the test. only config and data are needed

[the remark this answers is not in the document]
Comment (nicholasrq) [reply e2e/tests/empty-labels.test.js:188]: oops, i was supposed to remove the result as it's unused during the test. only config and data are needed

File: src/regions/RectRegion.js (hunk 1 above)
Comment (nicholasrq) [reply src/regions/RectRegion.js:22]: nah, was just observing api autocompletion and forgot to remove it

**Pull request merged**